.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+  - "6"
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
https://docs.travis-ci.com/user/languages/cpp/#C11-C%2B%2B11-(and-Beyond)-and-Toolchain-Versioning

In my experience, this is required for building binaries during `yarn` (maybe this is no longer required with trusty).